osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/infra/messaging/DlmsLogItemRequestMessageSender.java

```diff
@@ -31,46 +27,27 @@
   @Qualifier("protocolDlmsOutboundLogItemRequestsJmsTemplate")
   private JmsTemplate jmsTemplate;
 
-  public void send(final DlmsLogItemRequestMessage dlmsLogItemRequestMessage) {
-
-    LOGGER.debug("Sending DlmsLogItemRequestMessage");
+  @Value("${auditlogging.message.create.json:false}")
+  private boolean createJsonMessage;
 
-    this.jmsTemplate.send(new DlmsLogItemRequestMessageCreator(dlmsLogItemRequestMessage));
-  }
-
-  private static final class DlmsLogItemRequestMessageCreator implements MessageCreator {
+  public void send(final DlmsLogItemRequestMessage dlmsLogItemRequestMessage) {
+    if (dlmsLogItemRequestMessage == null) {
+      LOGGER.error("dlmsLogItemRequestMessage is null and will not be send");
```

Comment:
Separate the argument assertion from the logic dealing with proper input (see comments in other message sender classes for more detailed reasoning).